lxd/instance/drivers/driver_qemu.go

```diff
@@ -1018,8 +1018,8 @@ func (d *qemu) Start(stateful bool) error {
 	}
 
 	// Ensure secureboot is turned off for images that are not secureboot enabled
-	if d.localConfig["image.requirements.secureboot"] == "false" && shared.IsTrue(d.expandedConfig["security.secureboot"]) {
-		return errors.New("This image is not secure boot enabled")
+	if d.localConfig["image.requirements.secureboot"] == "false" && (d.expandedConfig["security.secureboot"] == "" || shared.IsTrue(d.expandedConfig["security.secureboot"])) {
```

Comment:
Err, secureboot logical expression was fixed too. The commit title is only telling half the story.

Reply:
Otherwise this should be good to go.